packages/redux-subspace-saga/src/sagas/subspaced.js

```diff
@@ -39,17 +39,19 @@ const subspaced = (mapState, namespace) => {
     const subspaceDecorator = subspace(mapState, namespace)
 
     return (saga) => {
-        return function* wrappedSaga() {
+        return function* wrappedSaga(...args) {
```

Comment:
Similar reasons to why we're [passing `args` through in the middleware](https://github.com/ioof-holdings/redux-subspace/pull/56/files#r158407618)